.devcontainer/Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM mcr.microsoft.com/vscode/devcontainers/rust:0-1
 
-# Update to Rust 1.58.0
-RUN rustup update 1.58.0 && rustup default 1.58.0
+# Update to Rust 1.58.1
+RUN rustup update 1.58.1 && rustup default 1.58.1
```

Comment:
Is this necessary anymore? Maybe we can remove this now that we have a toolchain file?

Reply:
yup, removed.